src/core/text/Text.js

```diff
@@ -354,6 +363,79 @@ export default class Text extends Sprite
     }
 
     /**
+     * Get trimmed transparent borders
+     *
+     * @private
+     * @returns {object} Trim data
+     */
+    getTrimmed()
+    {
+        // https://gist.github.com/remy/784508
+        const pixels = this.context.getImageData(0, 0, this.canvas.width, this.canvas.height);
+        const l = pixels.data.length;
+        const bound = {
+            top: null,
+            left: null,
+            right: null,
+            bottom: null,
+        };
+        let i;
+        let x;
+        let y;
+
+        for (i = 0; i < l; i += 4)
+        {
+            if (pixels.data[i + 3] !== 0)
+            {
+                x = (i / 4) % this.canvas.width;
+                y = ~~((i / 4) / this.canvas.width);
+
+                if (bound.top === null)
+                {
+                    bound.top = y;
+                }
+
+                if (bound.left === null)
+                {
+                    bound.left = x;
+                }
+                else if (x < bound.left)
+                {
+                    bound.left = x;
+                }
+
+                if (bound.right === null)
+                {
+                    bound.right = x + 1;
+                }
+                else if (bound.right < x)
+                {
+                    bound.right = x + 1;
+                }
+
+                if (bound.bottom === null)
+                {
+                    bound.bottom = y;
+                }
+                else if (bound.bottom < y)
+                {
+                    bound.bottom = y;
+                }
+            }
+        }
+
+        const trimHeight = bound.bottom - bound.top + 1;
+        const trimWidth = bound.right - bound.left;
+        const trimmed = this.context.getImageData(bound.left, bound.top, trimWidth, trimHeight);
+
+        return {
+            trimHeight,
+            trimWidth,
+            trimmed,
```

Comment:
Could you rename these output properties to `height`, `width` and `data`? The "trim" part of the names is implied.